pkg/reconcile/pipeline/infinispan/handler/manage/conditions.go

```diff
@@ -115,9 +114,18 @@ func AwaitWellFormedCondition(i *ispnv1.Infinispan, ctx pipeline.Context) {
 }
 
 func wellFormedCondition(i *ispnv1.Infinispan, ctx pipeline.Context, podList *corev1.PodList) ispnv1.InfinispanCondition {
+	// Avoid computation if CR requires CacheService when is forbidden
+	wellFormed := ispnv1.InfinispanCondition{Type: ispnv1.ConditionWellFormed}
```

Comment:
If we call `ctx.Stop` in `OperatorStatusChecks` then there's no need for this logic as it will never be reached.